mock-interactions.js

```diff
@@ -188,13 +244,14 @@
    * @param {HTMLElement} node The node to fire the event on.
    * @param {Object} xy Optional. The (x,y) coordinates the mouse event should be fired from.
    */
-  function tap(node) {
+  function tap(node, options) {
     // Respect nodes that are disabled in the UI.
     if (window.getComputedStyle(node)['pointer-events'] === 'none')
       return;
+
     var xy = middleOfNode(node);
-    down(node, xy);
-    up(node, xy);
+    down(node, xy, options);
```

Comment:
If emulateTouch is on, this would remove mousedown and mouseup right?

Even on a touch only phone, we should still get mousedown/mouseup/click events.